bin/install.sh

```diff
@@ -117,34 +145,77 @@ function _build_cli {
   done
 
   for (( i = 0; i < ${#sources[@]}; i++ )); do
-    # echo "$CXX" ${cflags[@]} -c "${sources[$i]}" -o "${outputs[$i]}"
     mkdir -p "$(dirname "${outputs[$i]}")"
     quiet "$CXX" "${cflags[@]}"  \
       -c "${sources[$i]}"        \
       -o "${outputs[$i]}"
-    die $? "not ok - unable to build. See trouble shooting guide in the README.md file"
+    die $? "not ok - unable to build. See trouble shooting guide in the README.md file:\n$CXX ${cflags[@]} -c \"${sources[$i]}\" -o \"${outputs[$i]}\""
   done
 
+  local exe=""
+  local libsocket_win=""
+  if [[ "$(uname -s)" == *"_NT"* ]]; then
+    exe=".exe"
+    libsocket_win="$BUILD_DIR/$arch-$platform/lib/libsocket-runtime.a"
+  fi
+
   quiet "$CXX"                                 \
     "$BUILD_DIR/$arch-$platform"/cli/*.o       \
     "${cflags[@]}" "${ldflags[@]}"             \
-    -o "$BUILD_DIR/$arch-$platform/bin/ssc"
+    "$libsocket_win"                           \
+    -o "$BUILD_DIR/$arch-$platform/bin/ssc$exe"
 
-  die $? "not ok - unable to build. See trouble shooting guide in the README.md file"
+  die $? "not ok - unable to build. See trouble shooting guide in the README.md file:\n$CXX ${cflags[@]} \"${ldflags[@]}\" -o \"$BUILD_DIR/$arch-$platform/bin/ssc\""
   echo "ok - built the cli for desktop"
 }
 
 function _build_runtime_library {
   echo "# building runtime library"
+  echo "$root/bin/build-runtime-library.sh" --arch "$(uname -m)" --platform desktop
   "$root/bin/build-runtime-library.sh" --arch "$(uname -m)" --platform desktop & pids+=($!)
-  if [[ "$HOST" = "Darwin" ]]; then
+  if [[ "$host" = "Darwin" ]]; then
     "$root/bin/build-runtime-library.sh" --arch "$(uname -m)" --platform ios & pids+=($!)
     "$root/bin/build-runtime-library.sh" --arch x86_64 --platform ios-simulator & pids+=($!)
   fi
 
   wait
 }
 
+function _get_web_view2() {
+  if [[ "$(uname -s)" != *"_NT"* ]]; then
+    return
+  fi
+  
+  local arch="$(uname -m)"
+  local platform="desktop"
+
+  if test -f "$BUILD_DIR/$arch-$platform/lib/WebView2LoaderStatic.lib"; then
+    echo "$BUILD_DIR/lib/$arch-$platform/WebView2LoaderStatic.lib exists."
+    return
+  fi
+
+  local tmp=$(mktemp -d)
+  local pwd=$(pwd)
+
+  echo "# Downloading Webview2"
+
+  curl -L https://www.nuget.org/api/v2/package/Microsoft.Web.WebView2/1.0.1619-prerelease --output $tmp/webview2.zip
```

Comment:
I thought about this, but recall that the goal here is to have one single code path.
Any duplicated code in `install.ps1` can be removed. 
I left the ps1 build process there for reference, but due to the fact that it can't work without implemented more duplicated code (Copying the `.a` etc), this code is basically vestigial at this point.

Note that we also have the same problem with `libuv` URI, which was actually coming from different sources in `install.ps1` and `install.sh`.

I just removed all unrequired code from `install.ps1` that I'm aware of.